UI/GameSettingsScreen.cpp

```diff
@@ -1156,6 +1156,21 @@ UI::EventReturn DeveloperToolsScreen::OnOpenTexturesIniFile(UI::EventParams &e)
 			if (f) {
 				fwrite("\xEF\xBB\xBF", 0, 3, f);
 				fclose(f);
+				// Let's also write some defaults
+				std::fstream fs;
+				File::OpenCPPFile(fs, texturesDirectory + "textures.ini", std::ios::out);
```

Comment:
This should be `std::ios::out | std::ios::ate`, so that it will append after the UTF-8 BOM written above.

-[Unknown]